OpenRA.Game/Server/Server.cs

```diff
@@ -1137,6 +1130,8 @@ public void SyncLobbyClients()
 
 				foreach (var t in serverTraits.WithInterface<INotifySyncLobbyInfo>())
 					t.LobbyInfoSynced(this);
+
+				pingUpdated.Restart();
```

Comment:
Maybe we should put a comment here that this needs to change if we ever implement sending delta-changes (like mentioned in the TODO above).